frontend/src/components/game/PeoplePicker.tsx

```diff
@@ -47,7 +50,8 @@ const Select: React.FC<PeoplePickerProps> = ({
                     people[0],
```

Comment:
What do you think about changing the onClick so it doesn't call onSelectPerson when disabled?